### ISSUE TITLE
Error message with b4

### ISSUE THREAD

**Issue by rileydog** [collective/collective.emailconfirmationregistration#7]
Hi,
   As mentioned in my other post, tool works great for sending link to user email to prevent spam registrations.  
## however, when the when the "review registration request" option is selected, get the following error:

Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module collective.emailconfirmationregistration.browser, line 359, in **call**
  Module z3c.form.form, line 218, in **call**
  Module plone.z3cform.fieldsets.extensible, line 59, in update
  Module plone.z3cform.patch, line 30, in GroupForm_update
  Module z3c.form.group, line 145, in update
  Module plone.app.z3cform.csrf, line 21, in execute
  Module z3c.form.action, line 98, in execute
  Module z3c.form.button, line 315, in **call**
  Module z3c.form.button, line 170, in **call**
  Module plone.app.users.browser.register, line 406, in action_join
  Module collective.emailconfirmationregistration.browser, line 314, in handle_join_success
  Module collective.emailconfirmationregistration.browser, line 345, in send_email_to_admin_to_review

**Comment by vangheem** [comment]
Want to give https://pypi.python.org/pypi/collective.emailconfirmationregistration/1.2.0b5 a try?

Thanks for testing.

**Comment by rileydog** [comment]
Hi,
tried it and works great.  
I turned on the 'review' and it worked properly.  When I registered to test, I got an email with a link, completed the username stuff, submitted, then got message it is under review (good, let's me know what's happening).  when, as admin, I approved the new member, I got another email with link, and went from there.
   I did see a request for the admin to get an email when a new review awaited.  This didn't happen..  Don't know if it was intended.  If this is not too difficult, it would be really important to an admin
   thanks for doing this.  I'd like to see you make it more widely known so more people use it.
